electrum/gui/kivy/uix/dialogs/lightning_open_channel.py

```diff
@@ -180,13 +182,16 @@ def open_channel(self):
         amount = '!' if self.is_max else self.app.get_amount(self.amount)
         self.dismiss()
         lnworker = self.app.wallet.lnworker
-        node_id, rest = extract_nodeid(conn_str)
-        if lnworker.has_conflicting_backup_with(node_id):
-            msg = messages.MGS_CONFLICTING_BACKUP_INSTANCE
-            d = Question(msg, lambda x: self._open_channel(x, conn_str, amount))
-            d.open()
-        else:
-            self._open_channel(True, conn_str, amount)
+        try:
+            node_id, rest = extract_nodeid(conn_str)
```

Comment:
if there is going to be a `ConnStringFormatError` raised, I think it is bound to happen already on this line?
I think we could restrict the try-except to only this line. What do you think? :)

Reply:
ok thanks, yes tried to save on the return statement, but it's better to keep the try-except clause as tight as possible I guess?

Reply:
I guess it is subjective... but yes, tight try-expect would be my preference.
Thanks!